eng/Directory.Build.Data.props

```diff
@@ -106,14 +106,17 @@
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <Authors>Microsoft</Authors>
     <Product>Azure .NET SDK</Product>
-    <PackageIcon>pkgicon.png</PackageIcon>
     <RepositoryUrl>https://github.com/Azure/azure-sdk-for-net</RepositoryUrl>
     <PackageProjectUrl>$(RepositoryUrl)</PackageProjectUrl>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <NeutralLanguage>en-US</NeutralLanguage>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(IsNetProject)' != 'true'">
```

Comment:
Instead of opting out I'd was hoping you could define a new property called `PackageIconPath` or something similar and set it to `$(RepoEngPath)/images/pkgicon.png` here and override it and PackageIcon in System.Memory.Data.csproj. 

Then where we include the icon in the package in the targets file just update that to use the `$(PackageIconPath)`.

Reply:
Would we need both packageIcon and packageIconPath?

Reply:
I guess packageIcon is a reserved property